discordbot/main.py

```diff
@@ -11,6 +11,10 @@
 
 intents = discord.Intents.default()
 intents.message_content = True
+intents.guilds = True
+intents.messages = True
+intents.reactions = True
```

Comment:
Lines 14, 15, 16 are redundant as these are included in discord.Intents.default().